#!/usr/bin/env python
PACKAGE = "dynamixel_controller"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("double_param_1", double_t, 0, "A double parameter to send",    -7, 0,   7)
gen.add("double_param_3", double_t, 0, "A double parameter to send",    -7, 0,   7)



exit(gen.generate(PACKAGE, "dynamixel_controller_node", "controller"))
